src/src/PluginStructs/P129_data_struct.cpp

```diff
@@ -36,8 +36,8 @@ bool P129_data_struct::plugin_init(struct EventStruct *event) {
   return false;
 }
 
-const uint32_t P129_data_struct::getChannelState(uint8_t offset,
-                                                 uint8_t size) const {
+uint32_t P129_data_struct::getChannelState(uint8_t offset,
```

Comment:
The const at the end was fine, as the function doesn't change a member.
So the function is const.
But the return value was just a deep-copy, so a const there is not making sense.

```
uint32_t P129_data_struct::getChannelState(...) const
```